converters/fromZigbee.js

```diff
@@ -1120,10 +1120,36 @@ const converters = {
         cluster: 'genLevelCtrl',
         type: ['commandMove', 'commandMoveWithOnOff'],
         convert: (model, msg, publish, options, meta) => {
+            const deviceID = msg.device.ieeeAddr;
+            if (!store[deviceID]) store[deviceID] = {};
+            if (!store[deviceID].timers) store[deviceID].timers = [];
+
             const direction = msg.data.movemode === 1 ? 'down' : 'up';
             const action = postfixWithEndpointName(`brightness_move_${direction}`, msg, model);
             const payload = {action, action_rate: msg.data.rate};
             addActionGroup(payload, msg, model);
+
+            if (options.hasOwnProperty('periodic_brightness_updates') &&
+            options.periodic_brightness_updates) {
+                let brightness = direction === 'up' ? 0 : 255;
+                const brightnessDelta = direction === 'up' ? 0.5 : -0.5;
```

Comment:
It might be nice to make this delta variable from the configuration file, that way the user can decide in which speed the brightness is adjusted when moving the controller :)

Reply:
Good idea :)
I have added an option for the delta variable. So now there are two ways to adjust the speed (update interval and delta value).